app/app.rb

```diff
@@ -0,0 +1,74 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setter'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  register Sinatra::Flash
+  use Rack::MethodOverride
+  set :session_secret, 'super secret'
+
+  get '/users/new' do
+    @user = User.new
+    erb :'users/new'
+  end
+
+  post '/sign-up' do
+    @user = User.new(email: params[:email], password: params[:password], password_confirmation: params[:password_confirmation], name: params[:name], user_name: params[:user_name])
```

Comment:
Metrics/LineLength: Line is too long. [179/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)